service/history/consts/const.go

```diff
@@ -0,0 +1,84 @@
+// The MIT License
```

Comment:
`const` is the same as `util`, `common`, and so one. I am against packages with such names. I usually put all errors to `errors.go` and all other consts to `defs.go` but leave them within the same package (`history` in this case).

If it causes cycles, then it is good sign that more refactoring is needed.

Reply:
1.  leaving the var definitions in history package will cause circular dependency. e.g. `history` package and `history/workflow` package will reference the same `history` package for errors, `history` package will reference `history/workflow` for internal logics
2. moving errors definitions into errors package will cause import conflict `"errors"` vs `"history/errors"`

let me know if your have other suggestions of names

Reply:
1. This is what I am saying. More refactoring and decomposition is needed here. I can't tell what exactly but this should be a ultimate goal here.
2. I am not suggesting to add `history/errors` package. I suggest to move errors definitions to `errors.go` within `history` package. And again, it probably will cause cycles, but see (1).

Reply:
1. the goal is to make the code structure easy to read / modify / understand, not to making the code shine.
2. like i said, there will be circular dependency issue, thanks to golang